packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -37,24 +40,30 @@ export const getTypeSymbol: unique symbol = Symbol("editable-tree:getType()");
 export const valueSymbol: unique symbol = Symbol("editable-tree:value");
 
 /**
- * A symbol to get the anchor of a node in contexts where string keys are already in use for fields.
+ * A symbol to insert a node in contexts where string keys are already in use for fields.
  */
-export const anchorSymbol: unique symbol = Symbol("editable-tree:anchor");
+export const insertNodeSymbol: unique symbol = Symbol("editable-tree:insertNode()");
 
 /**
- * A tree which can be traversed and edited.
- *
- * When iterating, only visits non-empty fields.
- * To discover empty fields, inspect the schema using {@link getTypeSymbol}.
+ * A symbol to set the value of a node in contexts where string keys are already in use for fields.
+ */
+export const setValueSymbol: unique symbol = Symbol("editable-tree:setValue()");
+
+/**
+ * A symbol to delete a node in contexts where string keys are already in use for fields.
+ */
+export const deleteNodeSymbol: unique symbol = Symbol("editable-tree:deleteNode()");
+
+/**
+ * A symbol to initialize a root node in contexts where string keys are already in use for fields.
+ */
+export const insertRootSymbol: unique symbol = Symbol("editable-tree:insertRoot()");
+
+/**
+ * {@link EditableTree}, but without fields i.e. having only utility symbols.
```

Comment:
Documenting a type in terms of a type that extends it (and thus you need to understand this to understand EditiableTree) create a cyclic dependency for people trying to understand this code.

its ok to mention EditableTree is a user of this, but understanding this type should not require understanding EditableTree

Reply:
Most probably I'll put it back into EditableTree definition, as it seems that EditableSequence should have completely different API.